src/camlsnark_c/libsnark-caml/libsnark/caml/caml_mnt4753.cpp

```diff
@@ -221,6 +221,10 @@ void camlsnark_mnt4753_field_mut_sub(FieldT* x, FieldT* y) {
   *x -= *y;
 }
 
+void camlsnark_mnt4753_field_copy(FieldT* x, FieldT* y) {
+  *x = *y;
```

Comment:
I'm not sure this is doing the copying correctly. I think it might be leaking the memory pointed to by x. The right thing to do would be to copy over the underlying buffer of one to the underlying buffer of the other.